src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -121,14 +121,30 @@ internal partial class Binder
             BindingDiagnosticBag diagnostics)
         {
             beforeAttributePartBound?.Invoke(node);
-            var boundAttribute = new ExecutableCodeBinder(node, this.ContainingMemberOrLambda, this).BindAttribute(node, boundAttributeType, (this as ContextualAttributeBinder)?.AttributedMember, diagnostics);
+            var boundAttribute = new ExecutableCodeBinder(node, this.ContainingMemberOrLambda, this).BindAttribute(node, boundAttributeType, (this as ContextualAttributeBinder)?.AttributeTarget, diagnostics);
             afterAttributePartBound?.Invoke(node);
             return (GetAttribute(boundAttribute, diagnostics), boundAttribute);
         }
 
-        internal BoundAttribute BindAttribute(AttributeSyntax node, NamedTypeSymbol attributeType, Symbol? attributedMember, BindingDiagnosticBag diagnostics)
+        internal BoundAttribute BindAttribute(AttributeSyntax node, NamedTypeSymbol attributeType, Symbol? attributeTarget, BindingDiagnosticBag diagnostics)
         {
-            return this.GetRequiredBinder(node).BindAttributeCore(node, attributeType, attributedMember, diagnostics);
+            return this.GetRequiredBinder(node).BindAttributeCore(node, attributeType, getAttributedMember(attributeTarget), diagnostics);
+
+            static Symbol? getAttributedMember(Symbol? symbol)
```

Comment:
I wanted to avoid changing `ContextualAttributeBinder.GetAttributedMember` from private to internal to call it in `AttributeSemanticModel`.
But it's okay either way. I'll change.